src/Microsoft.PowerShell.ConsoleHost/host/msh/Telemetry.cs

```diff
@@ -42,6 +43,28 @@ static ApplicationInsightsTelemetry()
             TelemetryConfiguration.Active.TelemetryChannel.DeveloperMode = _developerMode;
         }
 
+        internal static bool GetEnvironmentVariableAsBool(string name, bool defaultValue) {
```

Comment:
is there a reason for `internal` instead of `private`?  if you think this would be generally useful, perhaps this method should be `psutils.cs` instead of this file.  I would be ok with just making this private for now and in the future if this method is used elsewhere we can move it.

Reply:
To be honest, I think the `internal` came from dotnet-cli which is where I got the inspiration for this function. I have no rationale of choosing one over the other so I am happy with whichever you prefer.

Reply:
I would suggest leaving it here and changing to `private`.